src/motor-operations.h

```diff
@@ -64,22 +69,30 @@ class MotorOperations {  // Rename SegmentQueue ?
 
   // Wait, until all elements in the ring-buffer are consumed.
   virtual void WaitQueueEmpty() = 0;
+
+  // Get a reference of a FixedArray containing the absolute motors
```

Comment:
Here you talk about getting a reference, but instead, you return a value. Is the comment outdated or did you indeed meant to return a reference ?